README.md

```diff
@@ -1 +1,14 @@
-Orbit
+# Exposure System
+
+### Description
+
+Exposure system is a new initiative to expose **Humans of AltCampus** to various different knowledge other than just programming, by providing them with articles and tasks on a regular basis.
+
+This system also aims at bringing the people out of their comfort zone by giving them challenges, which would be in pairs or in groups. Completion of each of these challenges will result in positive incentives, while not participating will result in disincentives, motivating people to keep carrying on.
```

Comment:
@Yjspoder Instead write "As part of motivating people to do stuff the system incentivizes completion of these challenges and punishes people who don't."

Reply:
@prank7 Resolved